frontend/awx/resources/credentials/hooks/useCredentialActions.tsx

```diff
@@ -6,7 +6,7 @@ import { useNavigate } from 'react-router-dom';
 import { IPageAction, PageActionType } from '../../../../../framework';
 import { RouteObj } from '../../../../Routes';
 import { Credential } from '../../../interfaces/Credential';
-import { useDeleteCredentials } from '../useDeleteCredentials';
+import { useDeleteCredentials } from './useDeleteCredentials';
 
 export function useCredentialActions(options?: { onDeleted: (crednetials: Credential[]) => void }) {
   const { t } = useTranslation();
```

Comment:
I am able to view the edit and delete buttons in enabled state for a user with only read permissions to the credential.
I think we need to add the implementation for disabling or hiding the action buttons in this hook using the `isDisabled` field.